### ISSUE TITLE
nf-core lint should warn on non-standard Docker FROM instead of error

### ISSUE THREAD

**Issue by heuermh** [nf-core/tools#791]
```
╭──────────────────────────────────────────────────────────────────────────────╮
│ [✗] 1 Test Failed                                                            │
├──────────────────────────────────────────────────────────────────────────────┤
│ https://nf-co.re/errors#9: Could not find Dockerfile file string: FROM nfco… │
╰──────────────────────────────────────────────────────────────────────────────╯
```

https://github.com/nf-core/pangenome/runs/1439564894?check_suite_focus=true

**Comment by ewels** [comment]
Hi @heuermh,

I'm not sure that I totally agree here - why should it not fail? We're trying to avoid people using non-standard Docker containers, in theory so that we can easily add upstream fixes for standard problems across all pipelines in a single base location (has happened once or twice in the past).

If you need a specific Docker container for a given process then I think it might be better to build / use a process-specific container and use it only there, rather than messing with the container for the whole pipeline.

Note that we will remove this lint test completely pretty soon with the transition to DSL2.

Phil

**Comment by heuermh** [comment]
Yep, I agree with you.  Error means it fails our CI builds though, that's why I suggested warning instead.  We'll be migrating to full DSL2 as tools does, so this non-standard Docker FROM is only a temporary workaround.

**Comment by Zethson** [comment]
@heuermh @ewels I guess this issue will solve itself with the configurable linting discussed in #805 .

**Comment by ewels** [comment]
Configurable linting implemented in #809 so you will now be able to ignore this test if needed. Just make a root repo file `.nf-core-config.yml` with the following:

```yaml
conda_dockerfile: false
```

Available in v1.13 when it comes out.

Of course, a reminder that these lint tests exist for a reason.. 🙄 😅

**Comment by heuermh** [comment]
Thank you, @ewels!

> Of course, a reminder that these lint tests exist for a reason...

Agreed 😉
